scss/_o-navbar.scss

```diff
@@ -241,3 +241,49 @@
     align-self: center;
   }
 }
+
+.navbar-nav {
```

Comment:
I used as much existing utilities as possible, finally got it working with:
`<sup class="badge badge-info rounded-pill px-1 border border-bottom-0 border-info">5</sup>`.
Does it fit your requirements?

Reply:
To match dersign without heavy styles, I’m working with form with dark bakcground (`.bg-dark`) and added a custom border color with `.border-muted`.

I don’t know if it's good… Maybe a complete dark variant of `.form-control` would be better, wouldn't it?

Reply:
yes maybe need to think to a dark mode on that component. Because `border-muted` won't reproduce the focus color change applied on form component.
Need to see with @PigeardSylvain what are the design form forms component on dark background
Related to #117